generated/nidmm/nidmm.proto

```diff
@@ -53,7 +53,6 @@ service NiDmm {
   rpc ConfigureWaveformAcquisition(ConfigureWaveformAcquisitionRequest) returns (ConfigureWaveformAcquisitionResponse);
   rpc ConfigureWaveformCoupling(ConfigureWaveformCouplingRequest) returns (ConfigureWaveformCouplingResponse);
   rpc Control(ControlRequest) returns (ControlResponse);
-  rpc Control4022(Control4022Request) returns (Control4022Response);
```

Comment:
TODO: Figure out when we submit these changes how to remember to call out that we removed this function in the release notes.